ui/mirage/config.js

```diff
@@ -592,11 +592,29 @@ export default function () {
     return new Response(403, {}, null);
   });
 
+  this.get('/acl/roles', function ({ roles }, req) {
+    return this.serialize(roles.all());
+  });
+
```

Comment:
Everything under /mirage or /tests is not user-facing.

Mirage is our mocking library. We use this to build out demo environments (like on vercel), and to scaffold out test-cases for our acceptance tests.

Reply:
In particular, the methods you see listed here are meant to mock how Nomad's http api endpoints respond.